src/graphql/main/schema.graphql

```diff
@@ -309,6 +310,12 @@ type Coordinates {
   latitude: Float!
 }
 
+type MobileVersions {
+  platform: String!
+  currentSupported: Int!
```

Comment:
Should this be a `SemVer` string? Or is it common for mobile to represent versions as simple `Int`? What version is this actually referring too? Should the type be called `SuppertedMobileClientVersions`?

Reply:
Another thing that's not part of the core, but yeah thees are used in the mobile app to show that's an update is available or force an update.

These values are numbers on the backend.